script.skipit/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.skipit" name="SkipIt" version="1.0.0" provider-name="shartrec">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.python.script" library="addon.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en">Skip It</summary>
+		<description lang="en">Quick skip over Ads. This addon is based on the ideas from the original Topfield SkipIt TAP. It uses a binary search technique to quickly get you to the place you want in a video.</description>
+		<license>GNU General Public License, v2</license>
+		<language></language>
+<!-- 		<forum>http://forum.kodi.tv/showthread.php?tid=209948</forum> -->
+<!-- 		<source>https://github.com/zag2me/script.hello.world</source> -->
+<!-- 		<website>Kodi.tv</website> -->
+<!-- 		<email>zag@kodi.org</email> -->
+		<assets>
+			<icon>icon.png</icon>
```

Comment:
You have a screenshot in the submission but you don't include it in you add-on.xml. Also you are missing a fanart.

Reply:
Added fan art & updated addon.xml